### PR TITLE
Add target_type

### PR DESCRIPTION
**Pull request by dottorblaster**
Just adding target_type from the events and in the execution endpoint

### CODE REVIEW

File: test/wanda/executions/server_test.exs [hunk 1]
```diff
@@ -112,7 +112,7 @@ defmodule Wanda.Executions.ServerTest do
       )
 
       assert {:error, :already_running} =
-               Server.start_execution(UUID.uuid4(), group_id, targets, %{})
+               Server.start_execution(UUID.uuid4(), group_id, targets, :cluster, %{})
```
Comment (nelsonkopliku) [inline test/wanda/executions/server_test.exs:115]: we won't be getting atoms, rather strings. Just `"cluster"` should be enough.

**Review by nelsonkopliku — COMMENTED** [review]
Looking great, thanks!

Just a couple of comments.

**Review by nelsonkopliku — APPROVED** [review]
LGTM.
So if we get no target type upon request, then a `nil` is passed down to contract messages. It should not break anything with target type being a `string target_type`, right?

**Comment by dottorblaster** [comment]
It shouldn't, correct :+1:

**Pull request merged**